website/src/services/groupManagementApi.ts

```diff
@@ -18,6 +18,21 @@ const createGroupEndpoint = makeEndpoint({
     response: group,
     errors: [notAuthorizedError, conflictError],
 });
+const editGroupEndpoint = makeEndpoint({
+    method: 'put',
+    path: '/groups/:groupId',
+    alias: 'editGroup',
+    parameters: [
+        authorizationHeader,
+        {
+            name: 'data',
+            type: 'Body',
+            schema: newGroup,
+        },
+    ],
+    response: group,
+    errors: [notAuthorizedError], // TODO not sure which errors the API can throw
```

Comment:
We are not fully utilizing this `errors` feature. It's ok if this list is not exhaustive. It's good to include those that you care about though.

Reply:
Ok good to know thanks!